internal/services/containers/kubernetes_flux_configuration_resource_test.go

```diff
@@ -843,3 +906,59 @@ resource "azurerm_kubernetes_flux_configuration" "test" {
 }
 `, template, data.RandomInteger)
 }
+
+func (r KubernetesFluxConfigurationResource) gitRepositoryProviderAzure(data acceptance.TestData) string {
```

Comment:
Could we remove some of this code duplication by passing in the provider instead of having separate configs?